### PR TITLE
added a first change in two files

### PR DESCRIPTION
**Pull request by Sahith-4628**
made changes in the two files for the first time by adding one line in each

### CODE REVIEW

**Comment by Sahith-1379** [comment]
the first change in two files was very good

File: one.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 the original content of the first file
+
+first change
```
Comment (Sahith-1379) [inline one.txt:3]: add a second change after this line for first file

File: two.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 the original content of the second file
+
+first change
```
Comment (Sahith-1379) [inline two.txt:3]: add a second change after this line for second file

**Review by Sahith-1379 — CHANGES_REQUESTED** [review]
add second change, then will merge

File: one.txt (hunk 1 above)
Comment (Sahith-4628) [reply one.txt:3]: sure, ill add in a moment for the first file

File: two.txt (hunk 2 above)
Comment (Sahith-4628) [reply two.txt:3]: sure, ill add in a moment for second file

**Pull request closed without merging**